chihacknight/decarbonize-my-state
# re-added power generation data to pipeline, adds to state detail page

## Overview

For the power plant overview in https://github.com/chihacknight/decarbonize-my-state/issues/184#issuecomment-1430754415, this re-adds the power generation data by state from the EIA that @swatland610 originally prepped. This data includes data from 2001 to 2020. 

Grabs this data for a given state and makes the 2020 year available in a `console.log` statement:

```
all_solar: 506
coal: 31237.9
hydro_electric: 135
natural_gas: 24516.1
nuclear: 100246.1
petro_liquids: 30.3
wind: 16225.9
year: 2020
```

Not doing any data prep to determine how much is 'clean' vs other. We may want to do that work in the data prep script here: https://github.com/chihacknight/decarbonize-my-state/blob/main/data/scripts/utils.py#L49


## Testing Instructions

- visit a state detail page and confirm that there's a console statement showing power generation for 2020
